include/glow/Quantization/Base/Base.h

```diff
@@ -136,6 +136,14 @@ std::vector<int8_t> createMapping(TypeRef inTy, TypeRef outTy,
 void tensorRowwiseQuantization(const Tensor &input, Tensor &output,
                                Tensor &scales, Tensor &offsets);
 
+/// Fused-rowwise quantize the tensor \p input. Scales and offsets are generated
+/// from each row of \p input. \p output is tensor of the same shape as input
+/// but with 8 extra columns for storing fused scales and offset.
```

Comment:
maybe put a comment about the format here, something like data then 4 bytes for the scale then 4 bytes for offset